opm/autodiff/RateConverter.hpp

```diff
@@ -692,8 +692,19 @@ namespace Opm {
                 }
 
                 // Use average Rs and Rv:
-                double Rs = std::min(ra.rs, surface_rates[ig]/(surface_rates[io]+1.0e-15));
-                double Rv = std::min(ra.rv, surface_rates[io]/(surface_rates[ig]+1.0e-15));
+                auto a = ra.rs;
+                auto b = a;
+                if (io >= 0 && ig >= 0)
```

Comment:
If this is to be merged, it must at least emBRACE a few changes...

Reply:
oops, that opm-simulators convention is all-too easy to forget. done.